### ISSUE TITLE
beta4 installer offers silly options of where to install

### ISSUE THREAD

**Issue by tglsfdc** [XQuartz/XQuartz#87]
I just ran the 2.8.0_beta4 installer, and it offered me three choices of where to install to:

* Macintosh HD
* Time Machine Backups
* XQuartz-2.8.0_beta4

The Time Machine volume is mounted because a backup happened to be in progress.  I'm not sure whether offering to install there is insane or just dubious, but I am sure I've never seen an installer offer to do that before.  In any case, offering to install on the installer's own .dmg volume is certainly insane.

Big Sur 11.2.1 on an Intel Macbook.  I think I remember seeing beta3 also offer to install onto its .dmg.

**Comment by jeremyhu** [comment]
I suspect there's something I can set in the package distribution plist to hide this step...

**Comment by jeremyhu** [comment]
With this change, on Big Sur, "Destination Selection" is skipped but "Change Install Location..." is still present in the "Installation Type" section (<rdar://problem/74601724> filed for the inconsistency).  On 10.9, it seems this section is still not skipped.  I'm re-opening this to see if there is something else I can do here.

**Comment by tglsfdc** [comment]
Yeah, the strange behavior is still there in rc1 on Big Sur 11.2.1

**Comment by jeremyhu** [comment]
To be clear, there is a change in behavior.  The "Destination Selection" section is skipped, but the user is still given the option of going back in the "Installation Type" section.  I'm not sure if there is a way to disable that or not.

**Comment by jeremyhu** [comment]
rdar://95474325
